test/k8sT/Policies.go

```diff
@@ -155,7 +155,7 @@ var _ = Describe("K8sPolicyTest", func() {
 				By("HTTP connectivity to 1.1.1.1")
 				res := kubectl.ExecPodCmd(
 					namespaceForTest, pod,
-					helpers.CurlFail("http://1.1.1.1/"))
+					helpers.CurlFail("--retry 5 http://1.1.1.1/"))
```

Comment:
I think we shouldalso increase `CurlMaxTimeout` from `cons.go` here. I just realized, even with retry this will get interrupted by global Curl timeout.